src/map/pc.c

```diff
@@ -4180,16 +4195,41 @@ int pc_skill(struct map_session_data *sd, int id, int level, int flag)
 				status_calc_pc(sd, SCO_NONE);
 		break;
 		case SKILL_GRANT_TEMPORARY: //Item bonus skill.
```

Comment:
This part is bug, all item that give bonus skill show wrong skill value and also in window info